<!-- -->
<launch>

  <node name="h264_encoder_yolo" pkg="image_h264_encoder" type="image_h264_encoder_node" output="screen">
    <param name="fps" value="30" />
        <!-- <param name="bitrate" value="$(arg recording_bitrate)" /> -->
        <param name="decoderType" value="0" />
        <param name="logFilePath" value="/sampleData/h264Encoder_front.txt" />
        <param name="targetLocation" value="/sampleData/h264Encoder_front.mp4" />

        <remap from="~image" to="/VideoRaw" />
  </node>

  <!--node name="h264_encoder_depth" pkg="image_h264_encoder" type="image_h264_encoder_node" output="log">
    <param name="fps" value="20" />
    <param name="bitrate" value="10000" />
    <param name="decoderType" value="0" />
    <param name="logFilePath" value="/tmp/rec/h264EncoderDepth.txt" />
    <param name="targetLocation" value="/tmp/rec/h264Encoder10kDepth.mp4" />

    <param name="isDepthImage" value="True" />
    <param name="maxDepth" value="100" />

    <remap from="~image" to="/carla/ego_vehicle/camera/depth/mDepth/image_depth" />
  </node-->


</launch>
